### ISSUE TITLE
gen loss formula

### ISSUE THREAD

**Issue by edgarriba** [SKTBrain/DiscoGAN#1]
in lines [251-252](https://github.com/SKTBrain/DiscoGAN/blob/master/discogan/angle_pairing.py#L251-L252) you apply curriculum learning to compute the total gan loss. What's the motivation for that? Cannot find it in the paper.

Besides, is there any authorship relation between this repo and https://github.com/carpedm20/DiscoGAN-pytorch ?

**Comment by jazzsaxmafia** [comment]
We found out that in some cases assigning the same ratio to GAN loss and reconstruction loss fails to generate decent results. We guessed this occurred since training with GAN loss is much harder than training with reconstruction loss, especially in the early phase, and this curriculum strategy worked for those cases.

Carpedm20's code was independently implemented before we made this repo public, and his and ours have no authorship relation.

**Comment by edgarriba** [comment]
so then I guess that the formulas from the paper need a slight modification. Besides, `starting_rate` and `default_rate` have both the same value. Is it correct?

**Comment by hshyunsookim** [comment]
Thanks for pointing it out. We will consider making that change in our v2 paper. 

`starting_rate` and `default_rate` are equal only in `angle_pairing.py`.

**Comment by edgarriba** [comment]
great!
